lvmd/device_class_manager.go

```diff
@@ -36,6 +51,10 @@ type DeviceClass struct {
 	StripeSize string `json:"stripe-size"`
 	// LVCreateOptions are extra arguments to pass to lvcreate
 	LVCreateOptions []string `json:"lvcreate-options"`
+	// Type is the name of logical volume target, supports 'thick' (default) or 'thin' currently
+	Type LVType `json:"type,omitempty"`
```

Comment:
- in previous discussions it was made to respect the ThinPoolConfig if Type is `thick` or else ThinPoolConfig will not be used
- I also feel it's better to take decision based on Type, let me know otherwise on where this needs some more thought